apps/sms/js/thread_list_ui.js

```diff
@@ -498,7 +505,9 @@ var ThreadListUI = {
     var removeWhenEmpty = empty ? 'remove' : 'add';
 
     ThreadListUI.noMessages.classList[removeWhenEmpty]('hide');
+    ThreadListUI.settingsButton.classList[removeWhenEmpty]('hide');
     ThreadListUI.container.classList[addWhenEmpty]('hide');
+    ThreadListUI.optionsButton.classList[addWhenEmpty]('hide');
```

Comment:
how about toggling a new class "is-empty" on the "thread-list" container and changing the CSS accordingly?

Note that "classList.toggle" can take a second argument that is boolean, nowadays :)
```javascript
// use !! to force a boolean, because if it's undefined it will be as if it's absent.
panel.classList.toggle('threadlist-is-empty', !!empty);
```

@azasypkin maybe you'll have a better idea than me for the class name? I feel like `is-empty` is too small and thus too generic. But to be BEM-like we should add a "ThreadList" class and a "isEmpty" class to have a selector ".ThreadList.isEmpty". What do you think?